hls4ml/converters/keras/qkeras.py

```diff
@@ -80,15 +80,30 @@ def parse_qrnn_layer(keras_layer, input_names, input_shapes, data_reader):
     layer['weight_quantizer'] = get_quantizer_from_config(keras_layer, 'kernel')
     layer['recurrent_weight_quantizer'] = get_quantizer_from_config(keras_layer, 'recurrent')
     layer['bias_quantizer'] = get_quantizer_from_config(keras_layer, 'bias')
+    layer['accum_quantizer'] = get_quantizer_from_config(keras_layer, 'state')
+
+    if not isinstance(keras_layer['config']['activation'], str):
+        activation = get_activation_quantizer(keras_layer, input_names)
+
+        assert activation["class_name"] != "HardActivation", "Hard activation not supported"
+
+        layer['activation'] = activation['activation']
+        layer['activation_quantizer'] = activation['activation_quantizer']
+
+    if keras_layer['class_name'] in ['QLSTM', 'QGRU'] and not isinstance(keras_layer['config']['recurrent_activation'], str):
+        recurrent_activation = get_activation_quantizer(keras_layer, input_names, activation_name='recurrent_activation')
+
+        assert recurrent_activation["class_name"] != "HardActivation", "Hard activation not supported"
+
+        layer["recurrent_activation"] = recurrent_activation['recurrent_activation']
+        layer["recurrent_activation_config"] = recurrent_activation
 
     return layer, output_shape
 
 
-@keras_handler('QActivation')
```

Comment:
So what handles `QActivation` now? Otherwise, I like how the change is reused.

Reply:
The diff mangles the changes a git.
In this commit I have split the main functionality of parsing a `QActivation` into a new function:
```
def get_activation_quantizer(keras_layer, input_names, activation_name='activation'):
...
```

And QActivations are still handled by calling it:

```
@keras_handler('QActivation')
def parse_qactivation_layer(keras_layer, input_names, input_shapes, data_reader):
...
    layer = get_activation_quantizer(keras_layer, input_names)
...
```